### DISCUSSION TITLE
How exactly pricing works?

### DISCUSSION THREAD

**Discussion by ilbrigz** [deepgram/community#249] [General help] [answered]
Is the per minute pricing calculation rounded up to per request when the transcribe time is less than a minute?

My Transcription usage says I have sent around 91 requests which totals to 12 mins transcribe. For those transactions I have used up around .25usd of my credits. It seems the credits used and the total minute of transcribed audio does not sum up.

**Comment by jkroll-deepgram** [comment]
Hi @ilbrigz, you can see the exact cost of a request either in the Deepgram Console (Usage -> Logs), or through the [Usage API](https://developers.deepgram.com/reference/get-request) (`response["details"]["usd"]`). Also refer to our [Pricing](https://deepgram.com/pricing) page. Cost varies depending on the model and tier you're using, as well as if you're using audio intelligence add-ons such as summarization, entity detection, redaction, and topic detection. Notably, the cost is _not_ rounded up to the nearest minute - Deepgram bills per second of audio.

**Reply by ilbrigz** [reply]
![image](https://github.com/deepgram/community/assets/32309779/4b88a6ed-2b9c-49fb-963f-a4ec2f6c4252)
Hello @jkroll-deepgram. Can you explain how the cost was calculated in the attached example request? The audio length was only 26sec but the calculated cost is already 0.00542 which is more than the supposed $0.0044/min stated in your pricing page.

**Reply by jkroll-deepgram** [reply, marked as answer]
Hi @ilbrigz, I looked into this request ID you provided. I see you're using the `base` tier of the `general` model, which is $0.0125/minute, or $0.00542 for 26 seconds. If you switch to the `nova` tier (add `tier=nova` to your request parameters), then the pricing is $0.0044/minute, and the same request would be $0.00191 for the 26 seconds.

Nova offers the best speed, accuracy, and cost. Read more: [Introducing Nova: Next-Gen Speech-to-Text with Unmatched Performance](https://deepgram.com/learn/nova-speech-to-text-whisper-api)
